book/src/dev/README.md

```diff
@@ -138,6 +139,9 @@ Other:
 - `collisions`: an experimental data format for real-world collision data
 - `traffic_seitan`: a bug-finding tool that randomly generates live map edits
 - `tests`: integration tests
+- `experiment`: ...
```

Comment:
What's the experiment crate about?

Reply:
Deliberately vague. :D  I'll pitch my bizarre new idea offline.

Reply:
intrigue!